optuna_dashboard/ts/components/TrialTable.tsx

```diff
@@ -293,6 +333,47 @@ export const TrialTable: FC<{
             />
           </Box>
         </Grid>
+        {trials[index].state === ("Running" as TrialState) ? (
+          <Grid item xs={12}>
+            <Box margin={1}>
+              <Typography variant="h6" gutterBottom component="div">
+                Trial tell operations
+              </Typography>
+              <form onSubmit={handleSubmit}>
+                <Box margin={1}>
+                  <Stack direction="row" spacing={1}>
+                    {objectiveValues.map((value, i) => (
+                      <TextField
+                        id={`objective-${i}`}
+                        key={`objective-${i}`}
+                        label={`Objective ${i}`}
```

Comment:
How about displaying the `studyDetail.objectiveNames` defined by `set_objective_names()`? You can easily support implement it like the following code.

https://github.com/optuna/optuna-dashboard/blob/29233909a2d8ecba4b6e1a16f0cfdb9d76091d74/optuna_dashboard/ts/components/GraphParetoFront.tsx#L62-L64

Reply:
Fixed.